src/gcp_scanner/crawl.py

```diff
@@ -394,12 +394,12 @@ def get_bucket_names(project_name: str, credentials: Credentials,
 
 
 def get_managed_zones(project_name: str,
-                      credentials: Credentials) -> List[Dict[str, Any]]:
+                      service: discovery.Resource) -> List[Dict[str, Any]]:
   """Retrieve a list of DNS zones available in the project.
 
   Args:
     project_name: A name of a project to query info about.
-    credentials: An google.oauth2.credentials.Credentials object.
+    service: A resource object for interacting with the Compute API.
```

Comment:
This should say "DNS API", not "Compute API" right?

Reply:
yes, or can we say GCP API?